docs/resources/vpc.md

```diff
@@ -49,6 +49,12 @@ The following arguments are supported:
 * `description` - (Optional, String) Specifies supplementary information about the VPC. The value is a string of
   no more than 255 characters and cannot contain angle brackets (< or >).
 
+* `secondary_cidr` - (Optional, String) Specifies the secondary CIDR block of the VPC.
+
+  -> The argument cannot be imported into your Terraform state. And the following secondary CIDR blocks cannot be added
```

Comment:
suggest noting the argument which `cannot be imported `  in the import block

Reply:
fixed